document/core/syntax/modules.rst

```diff
@@ -66,13 +72,17 @@ Each class of definition has its own *index space*, as distinguished by the foll
    \production{table index} & \tableidx &::=& \u32 \\
    \production{memory index} & \memidx &::=& \u32 \\
    \production{global index} & \globalidx &::=& \u32 \\
+   \production{element index} & \elemidx &::=& \u32 \\
+   \production{data index} & \dataidx &::=& \u32 \\
    \production{local index} & \localidx &::=& \u32 \\
    \production{label index} & \labelidx &::=& \u32 \\
    \end{array}
 
 The index space for :ref:`functions <syntax-func>`, :ref:`tables <syntax-table>`, :ref:`memories <syntax-mem>` and :ref:`globals <syntax-global>` includes respective :ref:`imports <syntax-import>` declared in the same module.
 The indices of these imports precede the indices of other definitions in the same index space.
 
+The index spaces for :ref:`element segments <syntax-elem>` and :ref:`data segments <syntax-data>` are relative to a given :ref:`table <syntax-table>` and :ref:`memory <syntax-mem>` respectively.
```

Comment:
That is strange. Why would passive segments be associated with specific tables or memories?